### DISCUSSION TITLE
Character Shop

### DISCUSSION THREAD

**Discussion by EspectroW94** [AscensionGameDev/Intersect-Engine#1988] [Ideas]
It would be great to implement the character Shop system, leaving the character selling while it is afk.

something like this:

![Captura de pantalla 2023-10-18 224337](https://github.com/AscensionGameDev/Intersect-Engine/assets/127634629/5adedb95-5448-4bb4-a31e-406894bd2dc2)

In the screenshot you can see a character with a store "sign". When you enter it, it shows the items it sells and the price to pay.

**Comment by dhmello** [comment]
Man, if that were possible in this engine, I would even make a song for it... This system is simply widely used and would bring a lot of comfort to players, in addition to making the game have its own commerce, and not always depending on NPCS

**Reply by EspectroW94** [reply]
Besides, it would give more life to the game, since many of us leave the pjs selling while we go to work or do other things.

**Comment by PyroTech03** [comment]
I wonder if this is the one a dude is selling on itchio...

EDIT: grain of salt as the page seems iffy, but this is the system I'm talking about. With no images, demo, or contact info I'm leary to buy it at around $80 USD.

https://marmg.itch.io/patchmarket

**Reply by EspectroW94** [reply]
it seems fake

**Reply by PyroTech03** [reply]
Yea, seems sketchy as all hell lol.

**Comment by pandinocoder** [comment]
https://discord.com/channels/363106200243535872/965618005235081287/1171893515043741826

This won't be added officially but we could create a community-maintained unofficial patch
